dotCMS/src/integration-test/java/com/dotcms/auth/providers/jwt/services/JsonWebTokenServiceIntegrationTest.java

```diff
@@ -56,18 +56,26 @@ public static void prepare() throws Exception {
      * Testing the generateToken JsonWebTokenServiceTest
      */
     @Test
-    public void generateTokenTest() {
+    public void generateTokenTest() throws DotSecurityException, DotDataException {
 
+        final User user = APILocator.getUserAPI().loadUserById(userId);
         //Generate a new token
-        String jsonWebToken = jsonWebTokenService.generateUserToken(new UserToken(jwtId,
-                userId, new Date(), DateUtil.daysToMillis(2)));
+        final String jwtokenId    = user.getRememberMeToken();
+        final UserToken userToken = new UserToken.Builder()
+                .id(jwtokenId)
+                .subject(userId)
+                .issuer(clusterId)
+                .expiresDate(DateUtil.daysToMillis(2))
+                .claims(ImmutableMap.of())
+                .build();
+        String jsonWebToken = jsonWebTokenService.generateUserToken(userToken);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'jsonWebToken' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)